test/test_events.py

```diff
@@ -467,22 +445,29 @@ def test_broadcast_events(rabbit_manager, rabbit_config, start_containers):
 
 def test_requeue_on_error(rabbit_manager, rabbit_config, start_containers):
     vhost = rabbit_config['vhost']
-    start_containers(RequeueingHandler, ('requeue',))
+    container, = start_containers(RequeueingHandler, ('requeue',))
```

Comment:
i like brackets around this kind of tuple unpacking, for clarity

`(foo,) = ...`